plugin/helplink.vim

```diff
@@ -13,11 +13,7 @@ let g:loaded_helplink = 1
 let s:save_cpo = &cpo
 set cpo&vim
 
-augroup helplink.vim
-	autocmd!
-	autocmd Filetype help 
-		\ command! -buffer -nargs=* Helplink call s:echo(helplink#link(<q-args>))
-augroup end
+command! -nargs=* -complete=help Helplink call s:echo(helplink#link(<q-args>))
```

Comment:
This is just a personal preference, but I really like to keep my Vim commands as "clean" as possible by scoping it to specific buffers. I know I can `:delc Helplink` and then add the autocmd in my vimrc, but that's quite a lot of work.

On the other hand, I think that people wanting to have the command available everywhere can just call `command! ...`? That's simpler anyway. We could maybe make a helper function? `helplink#Command()` or some such.

I'd like to satisfy both preferences, but not sure if that's the best way?

Reply:
I see what you're saying. The main thing about a global command which I really like and that you can't directly get by just calling a function is the tab-completion. I can do `:Helplink ins-co<Tab>` and it gets completed to `:Helplink ins-completion`. For me, this is the primary way I want to use this plugin.

Perhaps it could be included globally if the user chooses, by setting a variable in their vimrc like

```vim
let g:helplink_global_command = 1
```